scala/src/main/scala/smile/clustering/distances/Binary/HammingDistance.scala

```diff
@@ -0,0 +1,15 @@
+package smile.clustering.distances
+
+class HammingDistance extends Binary_Distance
```

Comment:
No underscore in class name. `BinaryDistance` is preferred.

Reply:
Done